jest.config.js

```diff
@@ -6,10 +6,20 @@ module.exports = {
   moduleNameMapper: {
     '.(css|less|scss)$': 'identity-obj-proxy',
     '\\.svg$': '<rootDir>/__mocks__/svg.ts',
+    '\\.png$': '<rootDir>/__mocks__/svg.ts',
```

Comment:
`png.ts`
Not sure what these do, So please explain :)

Reply:
Should be 
```suggestion
    '\\.png$': '<rootDir>/__mocks__/png.ts',
```

It's (as far as I know) for mocking pictures in Jest.